app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+    @post = User.includes(:posts, :comments).find(params[:user_id]).posts.find(params[:post_id])
+    @comments = @post.most_recent_post
+  end
+
+  def create
+    parameters = comment_params
+    puts parameters
+    comment = Comment.new(post_id: params[:post_id], user_id: current_user.id, text: parameters[:text])
+    comment.save
+
+    if comment.save
+      redirect_to user_post_path(id: params[:post_id])
+    else
+      redirect_to new_user_post_comment
+    end
+  end
```

Comment:
A comment is created on behalf of the currently logged-in user (current_user) of the app 👌 Well done!